config.json

```diff
@@ -3755,6 +3755,34 @@
       }
     }
   },
+  "SOL": {
+    "code": "SOL",
+    "precision": 8,
+    "units": {
+      "SOL": {
+        "code": "SOL",
+        "symbol": "",
+        "name": "SOL",
+        "shift": 0,
+        "displayPrecision": 8,
+        "inputPrecision": 8
+      }
+    }
+  },
+  "USDC": {
+    "code": "USDC",
+    "precision": 5,
+    "units": {
+      "USDC": {
```

Comment:
one unit is enough? 

busd has also cents

Reply:
@ayrat555 for clarification, this was copy paste from here: https://github.com/coingaming/hub88-currency-config/commit/7d11dc925366cb7a2af9ef4a4d51b76d1eda8723